.travis.yml

```diff
@@ -10,17 +10,18 @@ jdk:
   - oraclejdk8
 
 before_install:
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
   - pip install --user codecov
 
 script:
-  - ./gradlew check
-  - ./gradlew codeCoverageReport
-  - ./gradlew sonarqube -PsonarLogin=$SONAR_TOKEN
+  - ./gradlew check --info -Pheadless
+  - ./gradlew codeCoverageReport -Pheadless=true
+  - ./gradlew sonarqube -PsonarLogin=$SONAR_TOKEN -Pheadless=true
 
 after_success:
   - codecov
-  - ./gradlew createHubOutput
-  - bash <(curl -s https://copilot.blackducksoftware.com/bash/travis) ./build/blackduck/*_bdio.jsonld
```

Comment:
Interesting the way this is marking the change, I thought master was already migrated?